packages/cosmos_ui_components/lib/components/template/transaction_history_card.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:cosmos_ui_components/cosmos_text_theme.dart';
+import 'package:cosmos_ui_components/cosmos_ui_components.dart';
+import 'package:cosmos_ui_components/utils/date_formatter.dart';
+import 'package:flutter/material.dart';
+
+class TransactionHistoryCard extends StatelessWidget {
+  const TransactionHistoryCard({
+    Key? key,
+    required this.isOutgoing,
+    required this.transactionType,
+    required this.date,
+    required this.amountText,
+    required this.denomText,
+  }) : super(key: key);
+
+  final bool isOutgoing;
+  final String transactionType;
+  final DateTime date;
+  final String amountText;
+  final String denomText;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = CosmosTheme.of(context);
+    return Padding(
+      padding: EdgeInsets.symmetric(horizontal: theme.spacingL, vertical: theme.spacingM),
+      child: Column(
+        children: [
+          Row(
+            children: [
+              Image.asset(
+                isOutgoing ? 'assets/images/send_icon.png' : 'assets/images/receive_icon.png',
+                package: packageName,
+              ),
+              SizedBox(width: theme.spacingL),
+              Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Text(
+                    transactionType,
+                    style: CosmosTextTheme.labelS,
+                  ),
+                  SizedBox(height: theme.spacingXS),
+                  Text(
+                    formatDate(date, DateFormatEnum.ShortUIDateDay),
+                    style: CosmosTextTheme.copyMinus1Normal,
+                  ),
+                ],
+              ),
+              const Spacer(),
+              Column(
+                crossAxisAlignment: CrossAxisAlignment.end,
+                children: [
+                  Text(
+                    '${isOutgoing ? '-' : '+'} $amountText',
+                    style: CosmosTextTheme.labelS.copyWith(color: isOutgoing ? null : Colors.green),
```

Comment:
hardcoded color

Reply:
The color for incoming transactions will always be green. Should I create a green color inside the theme?

Reply:
yes, we should always grab the colors from theme. especially that this should be `#008223` rather than `Colors.green`. Also, we can't assume that all implementations of the app will use green color for incomming transactions. some might theme it as blue for example, or use different shade of green for dark theme